src/py/flwr/common/constant.py

```diff
@@ -36,3 +36,12 @@
 MESSAGE_TYPE_GET_PARAMETERS = "get_parameters"
 MESSAGE_TYPE_FIT = "fit"
 MESSAGE_TYPE_EVALUATE = "evaluate"
+
+
+class MessageType:
+    """Message type."""
+
+    GET_PROPERTIES = MESSAGE_TYPE_GET_PROPERTIES
+    GET_PARAMETERS = MESSAGE_TYPE_GET_PARAMETERS
+    TRAIN = MESSAGE_TYPE_FIT
+    EVALUATE = MESSAGE_TYPE_EVALUATE
```

Comment:
why not define them here? instead of referencing those a few lines above?

Reply:
Many existing code still uses `MESSAGE_TYPE_*`, and thus I referencing the above ones to avoid divergence.

Reply:
The `MessageType` will be introduced in a separate PR to make this PR smaller.